styles/package.json

```diff
@@ -25,6 +25,5 @@
   },
   "publishConfig": {
     "access": "public"
-  },
-  "dependencies": {}
+  }
```

Comment:
This removed automatically when I ran `npm install`. I can revert this if it would cause any problems.